driver-core/src/test/unit/com/mongodb/operation/AsyncChangeStreamBatchCursorSpecification.groovy

```diff
@@ -46,13 +46,26 @@ class AsyncChangeStreamBatchCursorSpecification extends Specification {
         cursor.next(callback)
 
         then:
-        1 * wrapped.next(_) >> { it[0].onResult([], null) }
+        1 * wrapped.next(_) >> { it[0].onResult(null, null) }
```

Comment:
This change is not really related, but I saw when running the tests that there is an ugly stack track generated because of this, as the cursor doesn't expect an empty batch from the underlying AsyncQueryBatchCursor.  It should either be null or non-empty, and null serves well for the purpose of this test.